cts/scheduler/summary/asymmetrical-order-restart.summary

```diff
@@ -11,7 +11,7 @@ Online: [ cesr105-p16 cesr109-p16 ]
 Only 'private' parameters to cesr104ipmi_start_0 on cesr105-p16 changed: 0:0;1166:0:0:540ff5bf-81ee-4648-97cb-e922b82b370c
 Only 'private' parameters to cesr104ipmi_monitor_60000 on cesr105-p16 changed: 0:0;1167:0:0:540ff5bf-81ee-4648-97cb-e922b82b370c
 Transition Summary:
- * Stop       sleep_b         ( cesr109-p16 )   due to unrunnable sleep_a start
+ * Stop       sleep_b     ( cesr109-p16 )   due to unrunnable sleep_a start
```

Comment:
Crap.

Reply:
?
just whitespace

Reply:
Lol yeah, it's not broken (in this way). I didn't mean for the rebase push to add a bunch of whitespace changes to otherwise unchanged tests, is all.

Reply:
You already had a few before the rebase :)

Reply:
Ahhh, I must have `git-added` too many updated tests last night then. They weren't there before that.